GVFS/GVFS.Common/Http/HttpRequestor.cs

```diff
@@ -31,22 +41,51 @@ static HttpRequestor()
             availableConnections = new SemaphoreSlim(ServicePointManager.DefaultConnectionLimit);
         }
 
-        public HttpRequestor(ITracer tracer, RetryConfig retryConfig, GitAuthentication authentication)
+        protected HttpRequestor(ITracer tracer, RetryConfig retryConfig, Enlistment enlistment)

```

Comment:
Should the ssl settings be a property of the authentication object?

Reply:
Moved it there.